plugins/signal-protocol/CMakeLists.txt

```
@@ -105,7 +109,11 @@ set(CFLAGS ${VALA_CFLAGS} -I${CMAKE_CURRENT_SOURCE_DIR}/libsignal-protocol-c/src
 add_definitions(${CFLAGS})
 add_library(signal-protocol-vala ${SIGNAL_PROTOCOL_VALA_C} ${CMAKE_CURRENT_SOURCE_DIR}/src/signal_helper.c)
 add_dependencies(signal-protocol-vala signal-protocol-vapi)
-target_link_libraries(signal-protocol-vala ${SIGNAL_PROTOCOL_PACKAGES} gcrypt signal-protocol-c m)
+if(APPLE)
+    target_link_libraries(signal-protocol-vala ${SIGNAL_PROTOCOL_PACKAGES} signal-protocol-c m)
```

Add missing GCrypt library:
```suggestion
    target_link_libraries(signal-protocol-vala ${SIGNAL_PROTOCOL_PACKAGES} ${GCrypt_LIBRARY} signal-protocol-c m)
```

---

We should set the path via cmake variable. It so never a good Solution to use environment variables in not neccessary.
Dino makes use of cmake as a build environment, so we should use it.